core/src/components/item-divider/item-divider.scss

```
@@ -31,12 +31,19 @@
 
   @include font-smoothing();
   @include margin(0);
-  @include padding(
-    var(--padding-top),
-    var(--padding-end),
-    var(--padding-bottom),
-    calc(var(--padding-start) + var(--ion-safe-area-left, 0px))
-  );
+  @include padding(var(--padding-top), null, var(--padding-bottom));
```

Nit: Can we add `null` at the end? I know it doesn't make any functional difference, but it threw me off only seeing 3 values here.
```suggestion
  @include padding(var(--padding-top), null, var(--padding-bottom), null);
```